v2022.03.04-0.md (#722)
replicatedhq/kurl.sh: src/release-notes/v2022.03.04-0.md

---
date: "2022-03-04"
version: "v2022.03.04-0"
weight: 202203040
---

### <span class="label label-blue">Improvements</span>

> - Prevent envoy unresponsive behavior by adding CPU resource limits [Contour add-on](/docs/add-ons/contour) version 1.20.1.

Reviewer: I'd mention both requests and limits here IMO

Reviewer: ```suggestion
- Prevent envoy unresponsive behavior by adding CPU resource requests and limits to [Contour add-on](/docs/add-ons/contour) version 1.20.1+.
```

> - Prevent envoy unresponsive behavior by adding CPU resource requests and limits to [Contour add-on](/docs/add-ons/contour) version 1.20.1+.

Reviewer 2: @jala-dx Is this still accurate? "Envoy unresponsive behavior" isn't completely clear.
```suggestion
- Adds CPU resource requests and limits to the [Contour add-on](/docs/add-ons/contour) version 1.20.1+ to prevent Envoy from becoming unresponsive.
```
